 Header header
 sensor_msgs/Image[] data